src/components/ProgressBar/ProgressBar.props.ts

```diff
@@ -0,0 +1,14 @@
+export interface IProgressBarProps {
+    title: string;
+    width: number;
+    height: number;
+    percentage: string;
```

Comment:
i would really like two values. one for the current value, one for max value. And then the progress bar to automatically calculate the percentage. if i already have a percentage then i would just set the max value to 100.